src/extractors.ts

```diff
@@ -0,0 +1,43 @@
+import { Readability, isProbablyReaderable } from '@mozilla/readability';
```

Comment:
I would put all the extractor/util code that you took from other Actors into a folder so readers can easily see which code we might later want to put somewhere to import from (will handle the `google-extractors` library so it auto publishes)

Reply:
I move the relevant code to the `google-search` and `website-content-crawler` directories to improve the readability, please check in the branch [feat/create-extractor-dir](https://github.com/apify/rag-web-browser/tree/feat/create-extractor-dir/src)

Although, I'm not sure how it can be imported and auto-published in the future.

Reply:
For Google Search we have the package, I just didn't automate it yet https://www.npmjs.com/package/@apify/google-extractors

We could even update this Actor's repo and rebuild it automatically but it doesn't break as often so we will see how many users you will have.

Reply:
ok, thanks for the explanation

Reply:
FYI, the plan was to make this repo open source, hence it shouldn't depend on external private packages. For this use case (simply extracting titles + URLs from organic google search results), we thought simple CSS selector will do the job, without a need for the full-featured google extractor.

Reply:
@jancurn that's correct. Currently, we're not using the google-extractors library; instead, we're use css selectors to extract links from search results

Reply:
Yeah, that makes sense. The extractors broke like once or twice per year so you can manually update it based on our changes.

Reply:
Cool, the name of the directories confused me a bit :)